.github/workflows/benchmark-check.yml

```diff
@@ -0,0 +1,22 @@
+name: benchmarks checks
+on:
+  pull_request:
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with: { go-version: "1.20" }
+    - name: gobenchdata publish
+      run: go run go.bobheadxi.dev/gobenchdata@v1 action
```

Comment:
can you add a go install step before; just to make sure that we've more comprehensible error handling and sense of speed for install and run.